server/src/service.rs

```diff
@@ -318,6 +336,7 @@ impl Service {
         &self,
         locked: bool,
         objects: &[OwnedObjectPath],
+        prompt: bool,
```

Comment:
why is this needed?

Reply:
https://github.com/bilelmoussaoui/oo7/pull/173#discussion_r1966813203